.github/workflows/ci.yml

```diff
@@ -0,0 +1,288 @@
+            ~/.cargo/git
+          key: ${{ runner.os }}-cargo-rust_${{ matrix.rust }}-${{ hashFiles('**/Cargo.lock') }}
+      - name: Build cache
+        if: |
+          runner.os != 'macOS'
+        uses: actions/cache@v2
+        with:
+          path: target
+          key: ${{ runner.os }}-build-rust_${{ matrix.rust }}-target_${{ matrix.target }}-${{ hashFiles('**/Cargo.lock') }}
+      - name: Update deps
+        uses: actions-rs/cargo@v1
+        with:
+          command: update
+      - name: Build sys
+        uses: actions-rs/cargo@v1
+        env:
+          RUST_LOG: bindgen=warn,bindgen::ir=error,bindgen::codegen=error
+        with:
+          command: build
+          args: --manifest-path input-sys/Cargo.toml --target ${{ matrix.target }} --no-default-features --features update_bindings,libinput_1_11,libinput_1_14,libinput_1_15
+      - name: Upload bindings
+        if: matrix.task == 'bindings'
+        uses: actions/upload-artifact@v2
+        with:
+          name: bindings
+          path: |
+            input-sys/${{ env.INPUT_SYS_BINDINGS_FILE }}
+            LICENSE
+            README.md
+      - name: Build
+        uses: actions-rs/cargo@v1
+        with:
+          command: build
+          args: --target ${{ matrix.target }}
+      - name: Test
+        if: contains(matrix.target, '-linux-') && (startsWith(matrix.target, 'x86_64-') || startsWith(matrix.target, 'i686-'))
+        uses: actions-rs/cargo@v1
+        timeout-minutes: 12
+        env:
+          RUST_BACKTRACE: full
+        with:
+          command: test
+          args: --all --target ${{ matrix.target }}
+
+  update-bindings:
+    if: ${{ github.event_name != 'pull_request' && !startsWith(github.ref, 'refs/tags/') }}
+    needs:
+      - test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Download bindings
+        uses: actions/download-artifact@v2
+        with:
+          name: bindings
+      - name: Create pull request
+        uses: peter-evans/create-pull-request@v3
+        with:
+          base: ${{ github.head_ref }}
+          commit-message: Updated bindings
+          branch: update-bindings
+          delete-branch: true
+          title: Update bindings
+          body: |
+            Bindings should be updated to be consistent with latest changes
+
+  publish:
+    if: (github.repository == 'Smithay/input.rs' || github.repository == 'katyo/input.rs') && startsWith(github.ref, 'refs/tags/')
+    needs:
+      - format
+      - doc
+      - check
+      - test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+      - name: Setup Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          override: true
+      - name: Publish crates
+        uses: katyo/publish-crates@v1
+        with:
+          registry-token: ${{ secrets.CRATES_TOKEN }}
+          args: --no-verify
```

Comment:
Why is the `--no-verify` necessary?

Reply:
It is not necessary, but we already made much verifications above.